Updated code to work with Grunt 1.0.

radum opened the pull request:
I updated the code to make it work with the latest Grunt 1.0

Also Lodash is now a dependency as per Grunt best practices since 1.0

Commit:
Updated code to work with Grunt 1.0.

Commit:
Bump version

mikeapr4 commented:
Apart from the NPM errors, looks good to me

VeeTeeDev commented:
I did the same for the templating, but I moved the lodash to grunt itself because other dependencies are using it as well.

Can this be merged ? It's blocking :-)

ba55ie commented:
:+1:

Commit:
Update Travis and RequireJs dependency versions

Update Travis and RequireJs dependency versions

Update Travis and RequireJs dependency versions

Commit:
Update npm license warn, Lodash version

radum commented:
@mikeapr4 All good :D I was forgetting the Travis deps.

VeeTeeDev commented:
@cloudchen

radum commented:
Ping @cloudchen :)

Should we clone this if is not maintained :(

pksorensen commented:
what did you guys do ? cloned it to a new repository. no response from maintainers?

radum commented:
@pksorensen I'm using my own clone now. With the changes from this PR.

Commit:
Bump requirejs to 2.3.3 and grunt-contrib-jasmine

Commit:
Fix readme version changelog

Commit:
Fix the require.js path of the file by using the tempDir provided by grunt-contrib-jasmine

pranmitt commented:
are you going to merge this ?

Commit:
Added the repo to my npm org (radum)

Commit:
Added config for np npm publishing

Commit:
Update contributors package.json array

Commit:
0.3.3

Commit:
Updated requirejs ot 2.3.5 and added package-lock.json

Commit:
0.4.0

Commit:
Clean a bit jasmine-requirejs.html tpl

Commit:
0.4.1

Commit:
Update Travis badge

Commit:
Updated changelog list

Commit:
Updated readme maintainers list
